### ISSUE TITLE
SP doesn't support logout.

### ISSUE THREAD

**Issue by matejak** [timheap/flask-saml2#14]
I have set up a SP according to docs/example, and I have issue with logout.

Basically, I get a traceback caused by discrepancy between [logout call](https://github.com/timheap/flask-saml2/blob/master/flask_saml2/sp/views.py#L72) and [corresponding function definition](https://github.com/timheap/flask-saml2/blob/master/flask_saml2/sp/views.py#L74).

The failure looks legit.

**Comment by timheap** [comment]
You are correct, full SAML log out is not implemented yet. Currently, flask-saml2 can log you out of the SP by deleting the auth details stored in the session, then redirect you to the IdP's logout page which can also log you out by clearing the session. SAML Single Logout (SLO) is not implemented yet, from either the SP or IdP sides.

**Comment by raapidly** [comment]
can IdP be used in python3-saml?

**Comment by j00bar** [comment]
Is there an implementation approach you'd support around SLO? I imagine it would require some manner of shared storage that the `sp.get_auth_data_in_session` would have to check in with on every request, so that the session could be invalidated by the SLO endpoint?
